### PR TITLE
Renamed EfilingReviewerApiApplication to AIReviewerApiApplication

### PR DESCRIPTION
**Pull request by KDNeufeld**
# Description

This PR includes the following proposed change(s):

updated main launcher for AIReviewerApiApplication

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [x] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/ai-reviewer-api/pom.xml [hunk 1]
```diff
@@ -223,7 +223,7 @@
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
 				<configuration>
-					<mainClass>ca.bc.gov.open.jag.aireviewerapi.EfilingReviewerApiApplication</mainClass>
+					<mainClass>ca.bc.gov.open.jag.aireviewerapi.AIReviewerApiApplication</mainClass>
```
Comment (karthikmsqa) [inline src/backend/ai-reviewer-api/pom.xml:226]: Should 'AIReviewerApiApplication' be 'AiReviewerApiApplication'? (i.e. lower case 'i')

File: src/backend/ai-reviewer-api/src/main/java/ca/bc/gov/open/jag/aireviewerapi/AIReviewerApiApplication.java [hunk 2]
```diff
@@ -4,10 +4,10 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
-public class EfilingReviewerApiApplication {
+public class AIReviewerApiApplication {
```
Comment (karthikmsqa) [inline src/backend/ai-reviewer-api/src/main/java/ca/bc/gov/open/jag/aireviewerapi/AIReviewerApiApplication.java:7]: Same as above.

File: src/backend/ai-reviewer-api/src/main/java/ca/bc/gov/open/jag/aireviewerapi/AIReviewerApiApplication.java [hunk 3]
```diff
@@ -4,10 +4,10 @@
...
 
 	public static void main(String[] args) {
-		SpringApplication.run(EfilingReviewerApiApplication.class, args);
+		SpringApplication.run(AIReviewerApiApplication.class, args);
```
Comment (karthikmsqa) [inline src/backend/ai-reviewer-api/src/main/java/ca/bc/gov/open/jag/aireviewerapi/AIReviewerApiApplication.java:10]: Same as above.

File: src/backend/ai-reviewer-api/pom.xml (hunk 1 above)
Comment (KDNeufeld) [reply src/backend/ai-reviewer-api/pom.xml]: yeah, that's good. Updated name to AiReviewerApiApplication

File: src/backend/ai-reviewer-api/src/main/java/ca/bc/gov/open/jag/aireviewerapi/AIReviewerApiApplication.java (hunk 2 above)
Comment (KDNeufeld) [reply src/backend/ai-reviewer-api/src/main/java/ca/bc/gov/open/jag/aireviewerapi/AIReviewerApiApplication.java]: yeah, that's good. Updated name to AiReviewerApiApplication

File: src/backend/ai-reviewer-api/src/main/java/ca/bc/gov/open/jag/aireviewerapi/AIReviewerApiApplication.java (hunk 3 above)
Comment (KDNeufeld) [reply src/backend/ai-reviewer-api/src/main/java/ca/bc/gov/open/jag/aireviewerapi/AIReviewerApiApplication.java]: yeah, that's good. Updated name to AiReviewerApiApplication

**Review by karthikmsqa — APPROVED** [review]

**Pull request merged**